### ISSUE TITLE
Performance - setting / getting docs from ceramic node on 2019 mac takes 30+ seconds

### ISSUE THREAD

**Issue by Schwartz10** [ceramicnetwork/js-ceramic#519]
Been benchmarking our test cases, and ceramic slows each of our test cases down by (sometimes) 30+ seconds for 1-2 READ calls to the ceramic daemon. We're running ceramic-cli 0.13.3 and on 2019 macbook pros with sufficient memory.

**Comment by simonovic86** [comment]
@Schwartz10 can you try to run an IPFS instance separately from Ceramic? We created the repository for `js-ipfs-ceramic`:
* https://github.com/ceramicnetwork/js-ipfs-ceramic

Upon starting `js-ipfs-ceramic` you can connect to it like this:
```shell
ceramic daemon --ipfs-api=http://localhost:5011
```

My assumption is that the embedded IPFS is affecting the test cases.

**Comment by Schwartz10** [comment]
Hey @simonovic86 - this actually made things slower / worse / broken. In this case, I don't _think_ I've actually been able to successfully set _any_ docs (unless `setDIDProvider` under the hood is setting). Normally I'd think it was something with my IPFS setup, but reading IPFS data is working (when I tried shutting off access to port 5011, ceramic complains).

I've been waiting 10+ minutes to set 1 doc.
